.circleci/config.yml

```diff
@@ -566,18 +566,21 @@ jobs:
           command: |
             # Please see the runbook to determine when to update these values: https://fb.quip.com/FnSVAppvJaT9
             echo Premainnet git revision $PREMAINNET_GIT_REVISION
-            echo 'export NODES=a11d2e31854e811eaa1f702474484dd1-21dcb00bef9f6cc0.elb.us-west-2.amazonaws.com:80,\
-            a1981c0ab68b011eab1bb02f729e0dd7-3c9f20567ceb2efd.elb.us-west-2.amazonaws.com:80,\
-            a569b48655cbc40d092379f55e630966-e1f284c7eb8add73.elb.eu-west-1.amazonaws.com:80,\
-            aa4d66f81665240019022d50326c1654-cf61b8e99df12293.elb.eu-west-1.amazonaws.com:80,\
-            a88662827a2d341bbb7beb883d6e32b9-2e166f46672c7f2b.elb.us-east-1.amazonaws.com:80,\
+            echo 'export NODES=aa4d66f81665240019022d50326c1654-cf61b8e99df12293.elb.eu-west-1.amazonaws.com:80,\
+            aa82477d9b73b4531976e66cd4076eed-a1a497567aecf78e.elb.eu-west-1.amazonaws.com:80,\
```

Comment:
this one is not on the list either seems.

Reply:
can we add a comment on each node to indicate which company it's from? so it's easier to compare